packages/bsp/common/lib/systemd/system/armbian-disable-autologin.service

```diff
@@ -4,5 +4,6 @@ Description=Disable automated desktop login
 [Service]
 Type=simple
```

Comment:
probably change to oneshot and use multiple ExecStart commands instead of one ExecStart and one ExecStartPost.

Also add - before rm to make sure service is not marked as failed if the file doesn't exists